controllers/healthcheck_controller.go

```diff
@@ -1322,3 +1326,10 @@ func (r *HealthCheckReconciler) IsStorageError(err error) bool {
 	}
 	return false
 }
+
+func (r *HealthCheckReconciler) GetTimerByName(name string) *time.Timer {
+	r.Locker.RLock()
+	s := r.RepeatTimersByName[name]
```

Comment:
Do we also need to check existence of key here?
e.g.
```
r.Locker.RLock()
defer r.Locker.RUnlock()
ok, val := r.RepeatTimersByName[name]
if !ok {
  return nil
} else {
  return val
}
```

Reply:
Actually this will return nil if key is not found, so we should be good without this

Reply:
sure.